### DISCUSSION TITLE
A fault in glsp-elk-layout-engine

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-glsp/glsp#1373] [Q&A]
Hi @tortmayr
    There is a fault in  glsp-elk-layout-engine.ts, when the model is GLabel or GPort, the engine invokes transformPort. there is a transformLabel method in the engine never be invoked. is this an invoke fault? ,glsp version is 2.1.1
![image](https://github.com/user-attachments/assets/f56d132b-27b5-41ef-a11b-dba740805085)

**Comment by tortmayr** [comment]
Yes that definitly look like a bug to me.
Could you please open an issue for that? We have a [ milestone](https://github.com/eclipse-glsp/glsp/milestone/8) for working on bugs and improvements of the layout engine which we will work on in the next release cycle

**Reply by wss29** [reply]
Hi @tortmayr 
 I open an issue [1374](https://github.com/eclipse-glsp/glsp/issues/1374)
